tock/hours/views.py

```diff
@@ -925,7 +925,7 @@ def get_queryset(self, queryset=None):
         return sorted_fiscal_years
 
 
-class ReportingPeriodDetailView(ListView):
+class ReportingPeriodDetailView(PermissionMixin, ListView):
```

Comment:
Atul, this may be a terrible idea, but I wonder if it makes more sense to create an AuthListView that subclasses ListView (with the PermissionMixin), and we use that everywhere. Might be no better -- either way you'd have to remember to do _something_. But it's a thought.

Reply:
Seems like more indirection to me. But I do also thing that Tock is in a place to have an architectural discussion for both its current architecture and future iterations. 

@tbaxter-18f do you have hours to contribute to it? I could certainly use the help.

Reply:
I can find some time. I'm definitely not sure globally subclassing ListView would be any real benefit. Just something I was thinking about.